skype/skype-ps/skype/Get-CsOnlineUser.md

```diff
@@ -534,14 +576,105 @@ This cmdlet supports the common parameters: -Debug, -ErrorAction, -ErrorVariable
 
 ## INPUTS
 
-### Microsoft.Rtc.Management.ADConnect.Schema.OCSADUser or String
-A String must represent a valid user account Identity (for example, "sip:kenmyer@litwareinc.com").
+*Dropped input parameters*
+
+The following input paramters have beed dropped because they are no longer relevant to Teams:
+
+- ``[-LdapFilter <String>] ``
+- ``[-OnOfficeCommunicationServer] ``
+- ``[-OnModernServer]``
+- ``[-UnassignedUser] ``
+- ``[-OU <OUIdParameter>] ``
+- ``[-DomainController <Fqdn>]``
+
+*Dropped Filter operators*
+  
+The following filter syntaxes are not supported in Teams PowerShell Moduled 3.0.0 and above:
 
+- ``-not (<simple/complex PS filter>)``
+- ``<property> -like '*<text>'``
+- ``<property> -like '*<text>*'``
+- ``<property> -lt <value>``
+- ``<property> -gt <value>``
+- ``<PolicyPropertyName> -ge <value>, <PolicyPropertyName> -le <value>, <PolicyPropertyName> -gt <value>, <PolicyPropertyName> -lt <value>``
+
+*Supported filters*
+
+The Filtering functionality has been limited to the following attributes:
+
+- accountEnabled
+- ownerUrn
+- displayName
+- givenName
+- lineUri
+- userPrincipalName
+- ExternalAccessPolicy
+- OnlineDialOutPolicy
+- OnlineVoiceRoutingPolicy
+- TeamsMeetingPolicy
+- TeamsMeetingBroadcastPolicy
+- TeamsMessagingPolicy
+- TeamsCallParkPolicy
+- TeamsEmergencyCallingPolicy
+- TeamsEmergencyCallRoutingPolicy
+- TeamsChannelsPolicy
+- TeamsUpdateManagementPolicy
+- TeamsCallingPolicy
+- TeamsUpgradePolicy
+- TeamsUpgradeOverridePolicy
+- TeamsAppSetupPolicy
+- TeamsAppPermissionPolicy
+- TeamsVerticalPackagePolicy
+- TeamsSurvivableBranchAppliancePolicy
+- TeamsCallHoldPolicy
+- TenantDialPlan
+- OnlineVoicemailPolicy
+- OnlineAudioConferencingRoutingPolicy
+- TeamsAudioConferencingPolicy
+- TeamsVdiPolicy
+- TeamsFeedbackPolicy
+- TeamsIPPhonePolicy
+- TeamsShiftsAppPolicy
+- TeamsShiftsPolicy
+- TeamsTargetingPolicy
+- TeamsTemplatePermissionPolicy
+- TeamsSyntheticAutomatedCallPolicy
+- TeamsMobilityPolicy
+- TeamsCortanaPolicy
+- TeamsMeetingBrandingPolicy
+- TeamsNotificationAndFeedsPolicy
+- TeamsVideoInteropServicePolicy
+- TeamsEducationAssignmentsAppPolicy
+- TeamsComplianceRecordingPolicy
+- AssignedPlan
+- EnterpriseVoiceEnabled
+- Identity
+- department
+- UserDirSyncEnabled
+- Title
+- CountryAbbreviation
+- UsageLocation
+  
 
 ## OUTPUTS
 
-### Microsoft.Rtc.Management.ADConnect.Schema.ADOCOnlineUser
+*Attributes renamed:*
+
+- FirstName renamed to GivenName
+- DirSyncEnabled renamed to UserDirSyncEnabled
+- MCOValidationErrors renamed to UserValidationErrors
+- Enabled renamed to IsSipEnabled
+- TeamsBranchSurvivabilityPolicy renamed to TeamsSurvivableBranchAppliancePolicy
+
+*Attributes that have changed in meaning/format*
+
+OnPremLineURI: This attribute used to previously refer to both 
+  - LineURI set via OnPrem AD.
+  - Direct Routing numbers assigned to users via Set-CsUser.
```

Comment:
```suggestion
OnPremLineURI: This attribute used to refer to both:
  - Previous LineURI set via OnPrem AD.
  - Previous Direct Routing numbers assigned to users via Set-CsUser.
```

Reply:
@JohanFreelancer9 I had some additional changes which conflict with this commit. Please check now